### PR TITLE
MORE MORE MORE, I REQUIRE MORE CLOTHING

### PR DESCRIPTION
**Pull request by thestubborn**
from boh and bay, mostly
also some eyehud from main
whirrrrrr

oh yeah and i removed some references to >KHI
donate to my onlyfans

### CODE REVIEW

File: code/modules/clothing/glasses/hud_vr.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /obj/item/clothing/glasses/omnihud
 	name = "\improper AR glasses"
-	desc = "The KHI-62 AR Glasses are a design from Kitsuhana Heavy Industries. These are a cheap export version \
+	desc = "The KHI-62 AR Glasses are a design from Vey Med. These are a cheap export version \
```
Comment (Nethaufer) [inline code/modules/clothing/glasses/hud_vr.dm:3]: I would suggest changing KHI to another abbreviation like 'VM'. KHI as an acronym also means nothing in our lore.

Comment (Nethaufer) [reply code/modules/clothing/glasses/hud_vr.dm:3]: Same goes for the KHI in the other glasses you modified.

Comment (thestubborn) [reply code/modules/clothing/glasses/hud_vr.dm:3]: oh never even thought of that, will do

Comment (thestubborn) [reply code/modules/clothing/glasses/hud_vr.dm]: done btw

**Comment by thestubborn** [comment]
#1901

**Pull request closed without merging**